core-web/libs/sdk/analytics/README.md

```diff
@@ -0,0 +1,115 @@
+# @dotcms/analytics
+
+`@dotcms/analytics` is the official dotCMS JavaScript library that helps track events and analytics in your webapps. Currently available as an IIFE (Immediately Invoked Function Expression) module for direct browser usage, with planned future support for React and Next.js applications.
+
+## Features
+
+-   **Simple Browser Integration**: Easy to implement via script tags using IIFE implementation
+-   **Event Tracking**: Simple API to track custom events with additional properties
+-   **Automatic PageView**: Option to automatically track page views
+-   **Debug Mode**: Optional debug logging for development
+
+## Installation
+
+Include the script in your HTML page:
+
+```html
+<script src="analytics.iife.js"></script>
```

Comment:
How do I download this?

Reply:
By now, it is only used by DotCMS with auto-injection if you activate Content Analytics.